reconcile/utils/pagerduty_api.py

```diff
@@ -1,40 +1,78 @@
-import datetime
+from datetime import datetime as dt
+import logging
+from typing import Callable
+from typing import Iterable
+from typing import Optional
+from typing import Protocol
```

Comment:
nit: Callable, Iterable and Protocol should be imported from `collections.abc`

Reply:
aaahhh, not again 🤦 I really need an IDE utility for that ;)